src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -42,6 +42,21 @@ public class LanguageServer
 
         private TaskCompletionSource<bool> serverCompletedTask;
 
+        private static string checkHelpScript = @"
+[CmdletBinding()]
+param (
+    [String]$CommandName
+)
+Try {
+    Microsoft.PowerShell.Core\Get-Command $CommandName -ErrorAction Stop | Out-Null
```

Comment:
`Out-Null` as well :/ I know it's a pain but I'm pretty sure the last one we ran into was something like a different `Set-Content` that actually shipped with a product. I wouldn't be shocked if there was an `Out-Null` out there with mandatory parameters or something else breaking.

Or just do `$null = Microsoft.PowerShell.Core\Get-Command $CommandName -ErrorAction Stop`

Reply:
Copy pasta:
Yeah, I debated Out-Null or just assigning to $null. I think I recall reading somewhere that assignment is faster (but not by much, especially for this type of command)